crates/nu-cmd-extra/Cargo.toml

```diff
@@ -18,6 +18,8 @@ nu-parser = { path = "../nu-parser", version = "0.82.1" }
 nu-protocol = { path = "../nu-protocol", version = "0.82.1" }
 nu-cmd-base = { path = "../nu-cmd-base", version = "0.82.1" }
 nu-utils = { path = "../nu-utils", version = "0.82.1" }
+nu-command = { path = "../nu-command", version = "0.82.1" }
+nu-cmd-lang = { path = "../nu-cmd-lang", version = "0.82.1" }
```

Comment:
That is a nope! `nu-cmd-extra` should not regurlarly depend on them (only possibly as `dev-dependencies` if not circular)

Reply:
sheeeesh, according to `git blame`, this has been introduced in #9543 through the merge commit :thinking:

Reply:
it's like this on the `main` branch :eyes:

Reply:
[here](https://github.com/nushell/nushell/blob/main/crates/nu-cmd-base/Cargo.toml#L13), do you want me to change this in this PR? :thinking:

Reply:
#9543 is innocent: https://github.com/nushell/nushell/pull/9543/files#diff-111d1354eee39d38926ac7554b9802115710a8e6f5122d904adb1dab403cccb9

I remember I made an effort to isolate this in #9429: https://github.com/nushell/nushell/pull/9429/files#diff-111d1354eee39d38926ac7554b9802115710a8e6f5122d904adb1dab403cccb9

Reply:
i'm not sure i understand what should be done here if something needs to be addressed :confused:

Reply:
Those two lines should be gone (or only dev dependencies) and the crate should still compile.

Reply:
okey, i see, let me try that :yum: 

(btw, it was introduced in #9327, my bad...)

Reply:
it does fail with
```rust
error[E0433]: failed to resolve: use of undeclared crate or module `nu_command`
  --> crates/nu-cmd-extra/src/example_test.rs:64:43
   |
64 | ...dd_decl(Box::new(nu_command::Enumerate));
   |                     ^^^^^^^^^^ use of undeclared crate ormodule `nu_command`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `nu-cmd-extra` due to previous error
``` :eyes:

Reply:
iirc we need these to run the example tests of `each while` and `update cells` which require `enumerate` and `if` statement :thinking:

Reply:
Yeah but `example_test.rs` should be `#[cfg(test)]` only if I recall. So that would be fine as a dev dep.

Reply:
Thanks!

Reply:
:partying_face: